Samples/Samples.UWP/Package.appxmanifest

```diff
@@ -24,6 +24,7 @@
   </Applications>
   <Capabilities>
     <Capability Name="internetClient" />
+    <uap:Capability Name="appointments"/>
```

Comment:
does this need the uap prefix - just looking at consistency with the other capabilities

Reply:
Yeah I thought it was strange initially, but it just seems to be required for uwp appointments.

see: https://docs.microsoft.com/en-us/windows/uwp/packaging/app-capability-declarations
The appointments capability must include the uap namespace when you declare it in your app's package manifest as shown below.

<Capabilities><uap:Capability Name="appointments"/></Capabilities>